frontend/src/views/files/Listing.vue

```diff
@@ -394,6 +394,11 @@ export default {
 
           // Fill and fit the window with listing items
           this.fillWindow(true);
+          if (window.sessionStorage.getItem(this.$route.path)) {
+            document.documentElement.scrollTop = JSON.parse(window.sessionStorage.getItem(this.$route.path))[0];
+            if(!this.isMobile)  this.addSelected(JSON.parse(window.sessionStorage.getItem(this.$route.path))[1]);
```

Comment:
In iphone, when highlighting the item, we can see an obvious flash. It's a bug in iphone browser.  So just disable this highlight function in phone.